src/functions/helper_parser.ts

```diff
@@ -270,6 +268,11 @@ const TRANSLATED_CATEGORIES = {
   speed: _t("Speed"),
 };
 
+export const UNIT_OPTIONS = Object.entries(UNITS).map(([key, value]) => ({
+  value: key,
+  label: `${getTranslatedCategory(value.category)}`,
```

Comment:
this cast the translated string (`LazyTranslatedString`) to a real string, but translations are not loaded yet at the root of a file.
```suggestion
  label: getTranslatedCategory(value.category),
```